tools/pipelines-tasks/AVDAppAttachPublish/main.ts

```diff
@@ -79,11 +79,11 @@ async function run(): Promise<void> {
 
 		const powershellRunner: ToolRunner = helpers.getPowershellRunner(HELPER_SCRIPT);
 		powershellRunner.arg(['-inputJsonStr', '\'' + jsonString + '\'']);
-		powershellRunner.arg(['-targetDLL', TARGET_DLL]);
+		powershellRunner.arg(['-dllPath', TARGET_DLL_PATH]);
 
-		let execResult = await powershellRunner.execSync();
-		if (execResult.code) {
-			throw execResult.stderr;
```

Comment:
This refers to the error message as returned by App Atttach Framework so that user is clear of the reason for error. Can we have the same error reason even now ? Also, would recommended to completely verify the behavior for failure scnenario.

Reply:
In the PowerShell script we have the control over this.

I verified in some test scenarios, and I was able to see the error logs in the ADO screen, since it is handled from AppAttach backend logging mechanism. Let me know if anything more is needed.

Unfortunately, after changing function call to `await powershellRunner.exec();`, `stderr` is not supported for this variable anymore.